portfolio/src/main/webapp/script.js

```diff
@@ -103,25 +103,21 @@ function populateComments(commentSection) {
 
 function createComment(element) {
   const liComment = document.createElement('li');
-  liComment.innerText = element.key + ": " + element.value;
+  liComment.innerText = element[0] + ": " + element[1];
   liComment.className = "commentli";
-  liComment.id = element.key + element.value;
+  liComment.id = element[2];
+  console.log(liComment.id);
   const deleteComment = document.createElement('button');
   deleteComment.innerText = "X";
   deleteComment.className = "singleDelete";
   liComment.appendChild(deleteComment);
   deleteComment.addEventListener("click", function() {
-    deleteSingleComment(liComment.id);
+    deleteComments(liComment.id, "false");
```

Comment:
Why is `"false"` a string here?

Reply:
I added a parameter in the query url for whether to delete all comments, like you suggested, and I use a variable that is either "true" or "false." I guess I'm just a bit confused how to make the query string work if this value isn't a string? Sorry if I'm missing the point! I tried googling how to add a parameter to the query that isn't a string, but I'm not finding anything that is helping.

Reply:
Nevermind to my above comment, I already re-requested review by acciedent but I can change that now.

Reply:
Sorry to the above comment, I already re-requested review but I am working on this change now.

Reply:
I was asking because in index.html, you call `deleteComments(null,true)` (notice that the param is a boolean there, not a string).

It's better if you treat booleans like booleans in your code where possible. If you need to use strings for the query, that's fine. You should try to convert booleans to strings as late as possible in the process, so in your case it would be better for your `deleteComments` function to expect a `boolean` and then do the work of converting that to a string when it's building up the query URL.

Reply:
I understand, that makes sense. I was thinking about the query URL, and not how the value was stored before that - sorry!